src/lib/textToCad.ts

```diff
@@ -19,9 +19,13 @@ import { toSync } from './utils'
 
 async function submitTextToCadPrompt(
   prompt: string,
+  projectName: string,
   token?: string
 ): Promise<Models['TextToCad_type'] | Error> {
-  const body: Models['TextToCadCreateBody_type'] = { prompt }
+  const body: Models['TextToCadCreateBody_type'] = {
+    prompt,
+    project_name: projectName !== '' ? projectName : undefined,
```

Comment:
btw the body sent in the post request will be 
```
{
  prompt : <valueOfPrompt>,
  project_name : "undefined"
}
```

so you may actually end up parsing the project name as `undefined`.
If you want to NOT pass the key you should remove it from the `body` value.
does the post request need the `project_name`? 

This may not matter in this scenario but something to be aware of when writing a body for post request.

Reply:
oh annoying i figured it made it null, for real it turns it into a string?

Reply:
should i use null instead?

Reply:
either in rust will work out to the right thing, except if they are stringified lol

Reply:
okay i tested and this is working fine its not changing it to a string which is perfect

Reply:
the payloads look great in the network monitor dev tool